stage1/modules/linters-formatters-husky/README.md

```diff
@@ -0,0 +1,33 @@
+# [Stage#1.](../../) Linters, Formatters, and Husky
+
+## Objectives:
+
+- Get to know:
+  - Linters
+  - Formatters
+  - Husky
+- Explore reasons to use these tools
+- Get to know how to install and configure these tools 
+
+
+## Duration:
+Approximately 1,5 hour.
+
+## Theory:
+1. Watch [the video about linters](https://www.youtube.com/watch?v=HDQXWr5TOnI) (7 min)
+2. Read [the article about linters](https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/) (optional, ~ 7 min)
+3. Read [the article about Prettier](https://javascript.plainenglish.io/prettier-the-formatting-big-brother-of-eslint-2becf33168f9) (~ 5 min)
+4. Read [the article about Prettier and ESLint duo](https://blog.logrocket.com/using-prettier-eslint-automate-formatting-fixing-javascript/) (8 min)
+5. Watch [the video-tutorial how to work with ESLint and Prettier in VSCode](https://www.youtube.com/watch?v=St1YSNoB36Y) (~ 5 min)
+6. Read [docs about ESLint rules](https://eslint.org/docs/latest/use/configure/rules) (~7 min)
+7. Read [the article about husky](https://blog.bitsrc.io/ensure-javascript-code-quality-with-husky-and-hooks-6e338222662) (~ 5 min)
+8. Watch [the video-tutorial about husky setting up](https://www.youtube.com/watch?v=-qHA6zbOdLc) (~ 4 min)
+
+## Practice
+1. Students should take the "Linters, Formatters, and Husky" test in RS APP > Auto Test.
+
+## Additional materials:
+1. [Prettier documentation](https://prettier.io/)
+2. [ESLint configuration (docs)](https://eslint.org/docs/latest/use/configure/)
+3. [Popular style guides](https://blog.codacy.com/4-popular-javascript-style-guides)
+4. [Comparison of popular linters for JavaScript](https://www.sitepoint.com/comparison-javascript-linting-tools/) (optional)
```

Comment:
```suggestion
1. [Prettier documentation](https://prettier.io/)
2. [ESLint configuration (docs)](https://eslint.org/docs/latest/use/configure/)
3. [Popular style guides](https://blog.codacy.com/4-popular-javascript-style-guides)
4. [Comparison of popular linters for JavaScript](https://www.sitepoint.com/comparison-javascript-linting-tools/) (optional)
5. [Why Prettier](https://prettier.io/docs/en/why-prettier.html)
```

Reply:
Please add a link to the module [here](https://github.com/rolling-scopes-school/tasks/tree/master/stage1#january-2-2024) as well